config/fcos/v1_3/translate.go

```diff
@@ -64,6 +64,20 @@ func (c Config) ToIgn3_2Unvalidated(options common.TranslateOptions) (types.Conf
 		return types.Config{}, translate.TranslationSet{}, r
 	}
 	r.Merge(c.processBootDevice(&ret, &ts, options))
+	for i, disk := range ret.Storage.Disks {
+		// In the boot_device.mirror case, nothing specifies partition numbers
+		// so match existing partitions only when `wipeTable` is false
+		if (disk.WipeTable == nil) || (disk.WipeTable != nil && !*disk.WipeTable) {
```

Comment:
The second `WipeTable` check is redundant.

Reply:
...sorry, I phrased that poorly; all three of them are `WipeTable` checks.  I meant the second nil check.  So this should be:

```go
if disk.WipeTable == nil || !*disk.WipeTable {
```

which is actually just

```go
if !util.IsTrue(disk.WipeTable) {
```

Reply:
Fix in https://github.com/coreos/butane/pull/288.